rbtree.h

```diff
@@ -19,7 +19,7 @@
   linux/include/linux/rbtree.h
 
   To use rbtrees you'll have to implement your own insert and search cores.
-  This will avoid us to use callbacks and to drop drammatically performances.
+  This will avoid us to use callbacks and to drop dramatically performances.
```

Comment:
I think this needs a grammar check too. Maybe the version in current Linux kenel has it better worded.